tools/codegen/cmd/root.go

```diff
@@ -90,7 +91,7 @@ func executeGenerators(genCtx generation.Context, generators ...generation.Gener
 	}
 
 	if len(errs) > 0 {
-		return kerrors.NewAggregate(errs)
+		return utils.NewAggregate(kerrors.NewAggregate(errs))
```

Comment:
nit: Does `utils.Aggregate` implements the `Aggregate` interface? If not, this name might be a bit misleading.